src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -123,7 +140,16 @@ def get_helper_scores(self, text_input: List[str]) -> Dict[str, List[float]]:  #
         :param text_input: list of text inputs for the model
         :returns: dict with keys as score name and value being list of scores for text inputs
         """
-        return self._model(text_input)
+        inputs = self._tokenizer(text_input, return_tensors="pt", truncation=True, padding=True).to(self._model.device)
+        scores = torch.sigmoid(self._model(**inputs)[0]).cpu().detach().numpy()
+        results = {}
+        for i, cla in enumerate(DetoxifyHelperModel.get_score_names()):
+            results[cla] = (
+                scores[0][i]
+                if isinstance(text_input, str)
```

Comment:
indeed, from signature text_input should be a List. I don't think this method works if text_input is a list with more than one string (because of line 144)

Reply:
See comment above.